### DISCUSSION TITLE
expunged with first use

### DISCUSSION THREAD

**Discussion by tomkrahn** [vzhd1701/evernote-backup#74] [Q&A]
The first time I ran evernote-backup, I received the message that many notes were expunged.  (more than were in the evernote trash bin).
What notes are expunged, and where are they expunged from?  Does evernote-backup change anything on the evernote side of things?
Tom

**Comment by tomkrahn** [comment]
I've continued using the program.  Today it said that 392 notes were updated.  I'm sure that I haven't changed more than 25 notes.  What's up with that?
Tom

**Comment by vzhd1701** [comment]
Expunged means that Evernote server sends a list of notes that are no longer there and evernote-backup removes them from it's internal database. They may be notes that are long gone, but counted anyway. evernote-backup does not change anything, it's a read only tool.

Also Evernote can change something internally on server and mark notes for re-sync.
